src/System.Net.Primitives/src/System/Net/IPAddress.cs

```diff
@@ -150,6 +150,30 @@ public IPAddress(byte[] address, long scopeid)
             PrivateScopeId = (uint)scopeid;
         }
```

Comment:
I didn't think of throwing the exception that way. Started with sharing the code but then the null check had to be specifically in the array based ctor and I ended up with the copy pasting of code.

Will change both ctors to use this type of null check. Thanks!